packages/connectivity/src/scp-cf/destination/destination-from-registration.spec.ts

```diff
@@ -116,18 +116,6 @@ describe('register-destination', () => {
     ).resolves.toBe(true);
   });
 
-  it('caches with unlimited time', async () => {
```

Comment:
[req] I think we still need this test. The register destination cache should never expire by default, but as far as I can see there is no such option currently possible.

The ticket for this says "Align on a parameter that reflects the value "never" (ideas: "never", null, undefined)" => I think `undefined` is not a good choice because it is not explicit enough. I would maybe consider `0` because this value does not make sense as a default and could signify that it never expired. I think I saw this in other APIs as well.

Reply:
I've now re-enabled this test. For unlimited expiration time, I went with your suggestion to use `0` to signify "never expired".